docs/porting/configuration-differences.md

```diff
@@ -0,0 +1,597 @@
+  </td>
+ </tr>
+ <tr>
+  <td rowspan=5>
+  <p>
+    <span>
+      <a href="https://www.openssl.org/docs/manmaster/man3/SSL_CTX_set_mode.html">
+        SSL_CTX_set_mode<br>
+        SSL_set_mode
+      </a>
+      <br><br>
+      <a href="https://www.openssl.org/docs/manmaster/man3/SSL_CTX_set_mode.html">
+        SSL_CTX_clear_mode<br>
+        SSL_clear_mode
+      </a>
+    </span>
+  </p>
+  </td>
+  <td>
+  <p>
+    <span>
+      <a href="https://github.com/aws/aws-lc/blob/c8d82c7599449609d3540eefb7972f137fc1b872/include/openssl/ssl.h#L839-L849">
+        SSL_MODE_NO_AUTO_CHAIN
+      </a>
+    </span>
+    </p>
+  </td>
+  <td>
+  <p><span>ON</span></p>
+  </td>
+  <td>
+  <p><span>Configurable</span></p>
+  </td>
+ </tr>
+ <tr>
+  <td>
+  <p><span>SSL_MODE_AUTO_RETRY</span></p>
+  </td>
+  <td>
+  <p><span>ON</span></p>
+  </td>
+  <td>
+  <p><span>NO-OP</span></p>
+  </td>
+ </tr>
+ <tr>
+  <td>
+  <p><span>SSL_MODE_RELEASE_BUFFERS</span></p>
+  </td>
+  <td>
+  <p><span>ON</span></p>
+  </td>
+  <td>
+  <p><span>NO-OP</span></p>
+  </td>
+ </tr>
+ <tr>
+  <td>
+  <p><span>SSL_MODE_SEND_CLIENTHELLO_TIME</span></p>
+  </td>
+  <td>
+  <p><span>OFF</span></p>
+  </td>
+  <td>
+  <p><span>NO-OP</span></p>
+  </td>
+ </tr>
+ <tr>
+  <td>
+  <p><span>SSL_MODE_SEND_SERVERHELLO_TIME</span></p>
+  </td>
+  <td>
+  <p><span>ON</span></p>
+  </td>
+  <td>
+  <p><span>NO-OP</span></p>
+  </td>
+ </tr>
+ <tr>
+  <td rowspan=10>
+  <p>
+    <span>
+      <a href="https://www.openssl.org/docs/manmaster/man3/SSL_CTX_set_options.html">
+        SSL_CTX_set_options<br>
+        SSL_set_options
+      </a>
+      <br><br>
+      <a href="https://www.openssl.org/docs/manmaster/man3/SSL_CTX_set_options.html">
+        SSL_CTX_clear_options<br>
+        SSL_clear_options
+      </a>
+    </span>
+    </p>
+  </td>
+  <td>
+  <p><span>SSL_OP_ALL</span></p>
+  </td>
+  <td>
+  <p><span>OFF</span></p>
+  </td>
+  <td>
+  <p><span>NO-OP</span></p>
+  </td>
+ </tr>
+ <tr>
+  <td>
+  <p><span>SSL_OP_ALLOW_UNSAFE_LEGACY_RENEGOTIATION</span></p>
+  </td>
+  <td>
+  <p><span>OFF</span></p>
+  </td>
+  <td>
+  <p><span>NO-OP</span></p>
+  </td>
+ </tr>
+ <tr>
+  <td>
+  <p><span>SSL_OP_DONT_INSERT_EMPTY_FRAGMENTS</span></p>
+  </td>
+  <td>
+  <p><span>ON</span></p>
+  </td>
+  <td>
+  <p><span>NO-OP</span></p>
+  </td>
+ </tr>
+ <tr>
+  <td>
+  <p><span>SSL_OP_LEGACY_SERVER_CONNECT</span></p>
+  </td>
+  <td>
+  <p><span>OFF</span></p>
+  </td>
+  <td>
+  <p><span>NO-OP</span></p>
+  </td>
+ </tr>
+ <tr>
+  <td>
+  <p><span>SSL_OP_NO_COMPRESSION</span></p>
+  </td>
+  <td>
+  <p><span>ON</span></p>
+  </td>
+  <td>
+  <p><span>NO-OP</span></p>
+  </td>
+ </tr>
+ <tr>
+  <td>
+  <p><span>SSL_OP_NO_RENEGOTIATION</span></p>
+  </td>
+  <td>
+  <p><span>ON</span></p>
+  </td>
+  <td>
+  <p><span>NO-OP<br>
+  <br>
+  Renegotiation is enabled with </span><span style='font-size:10.0pt;
+  font-family:"Courier New"'>SSL_set_renegotiate_mode</span>
```

Comment:
In github's rendering, I don't see SSL_set_renegotiate_mode as Courier New.
The comma following it is at the beginning of the next line (maybe my resolution setting, but it's not "glued" to it).

Reply:
Ah sure, I don’t think markdown’s html renders specific fonts, so the “Courier New” style is a bit redundant. I’ll remove them and tweak the commas while I'm at it.